containerd/README.md

```diff
@@ -35,12 +35,20 @@ spec:
           volumeMounts:
             - name: containerdsocket
               mountPath: /var/run/containerd/containerd.sock
+            - mountPath: /host/var/run
+              name: var-run
+              readOnly: true
           volumes:
             - hostPath:
                 path: /var/run/containerd/containerd.sock
               name: containerdsocket
+            - hostPath:
+                path: /var/run
+              type: DirectoryOrCreate
```

Comment:
we should remove `type: DirectoryOrCreate` as thisshould already exist on the host

Reply:
👍